open_in_mpv/open_in_mpv.py

```diff
@@ -189,10 +189,13 @@ def real_main(log: TextIO) -> int:
         response(dict(version=VERSION, logPath=log.name, socketPath=MPV_SOCKET))
         log.close()
         return 0
-    if (url := message.get('url', None) is None):
+    if (url := message.get('url', None)) is None:
         logger.exception('No URL was given')
         print(json.dumps(dict(message='Missing URL!')))
         return 1
+    if 'https' not in url:
+        print(json.dumps(dict(message='Security blocked unsecure connection.')))
```

Comment:
Make it say: *Insecure URLs (non-HTTPS) are blocked by default.*

Reply:
Done.